test/index.js

```diff
@@ -0,0 +1,69 @@
+var test = require('tape');
+var CoinsLogonWidget = require('../scripts/coins-logon-widget.js');
+var html = require('html!./index.html');
+var jQuery = require('jquery');
+var Promise = this.Promise = require('bluebird'); // phantomJS polyfill. seriously. :-|
+jQuery(window.document.body).append(html);
+
+var server;
+
+var setWidgetFields = function(opts) {
+    var widget = opts.widget;
+    widget.form.formGroups[0].element.children[1].value = opts.username;
+    widget.form.formGroups[1].element.children[1].value = opts.password;
+};
+
+test('constructor, basic', function(t) {
+    var el = document.getElementById('regular_form');
+    var myWidget = new CoinsLogonWidget({
+        el: el,
+        baseUrl: 'coins-api-root',
+        authCookieName: 'coins-cookie-nom-noms',
+    });
+    t.ok(myWidget, 'constructor returns a widget');
+    t.equal(myWidget.element, el, 'constructor sets el');
+    t.ok(jQuery('#regular_form:visible').length, 'form visible on DOM');
```

Comment:
This checks that `<div id="regular_form">...</div>` is visible. Change to `#regular_form .coins-logon-widget:visible` or `#regular_form > *:visible`?